server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -84,6 +84,19 @@ export class RwcdsFormService {
   }
 
   async find(id) {
+    // Because we post data from dcp_projectaction to dcp_affectedzoningresolution, we need
+    // to GET the rwcdsForm twice. Once to pass the form into the syncActions method. And 
+    // a second time to include the updated dcp_affectedzoningresolution relationship. 
+    const { records: [rwcdsFormWithoutUpdatedZr] } = await this.crmService.get(`dcp_rwcdsforms`, `
+      $filter=
+        dcp_rwcdsformid eq ${id}
+      &$expand=dcp_rwcdsform_dcp_affectedzoningresolution_rwcdsform
```

Comment:
It looks like you can delete the expand here, since the second rwcdsForm GET also expands `dcp_rwcdsform_dcp_affectedzoningresolution_rwcdsform`. The first expand of affectedzoningresolution is to support creating `zrTypes` within `syncActions()` , but I think you can just use the expand from the second GET.

Reply:
But don't we need the first `expand` to be able to access `dcp_affectedzoningresolution` in order to pass it into `syncActions`

Reply:
Thanks for clarifying this during standup! I think I didn't have enough coffee on Thursday.